trie/constants.py

```diff
@@ -34,3 +34,10 @@
 
 BYTE_1 = bytes([1])
 BYTE_0 = bytes([0])
+
+# Constants for Sparse Merkle Tree
+from eth_hash.auto import keccak
+EMPTY_LEAF_NODE_HASH = BLANK_HASH
+EMPTY_NODE_HASHES = [EMPTY_LEAF_NODE_HASH]
+for _ in range(159):
```

Comment:
As 160 (and 159) is used a couple of times, it would make sense to make it a constant. Maybe `NODE_BIT_LENGTH`?

Reply:
Agree, what do you think about maybe something more straight forward like `TREE_HEIGHT`?

Reply:
`TREE_HEIGHT` sounds good